Readme.md

```diff
@@ -165,7 +164,7 @@ const demo1 = async connection => {
     },
   })
 
-  const result = await me.execute(connection)
+  const result = await connection.execute([me])

```

Comment:
I'm a little surprised that this would be necessary. I'd expect the former to work?

and it's a little bit more sugary. `connection.execute` handle an array of invocations, but the `me.execute` only executes the `me` invocation (api doesn't involve any array)

Reply:
it was crashing the server with a car parse error

Reply:
@jchris I'll try to watch for that, but if you repro it again, send me the error stack trace

Reply:
example of how we use `IssuedInvocation#execute` in the wild https://github.com/web3-storage/w3protocol/blob/fc53a16c40f1558b2b17dd4ff9e301a1ef54950f/packages/access-api/test/space-info.test.js#L27